packages/content-fetch/src/job.ts

```diff
@@ -48,6 +48,10 @@ const getOpts = (job: savePageJob): BulkJobOptions => {
     // removeOnFail: true,
     attempts: getAttempts(job),
     priority: getPriority(job),
+    backoff: {
+      type: 'exponential',
+      delay: 1000,
```

Comment:
should we also change `delay` to `2000`?

Reply:
agreed!